lib/venue.rb

```diff
@@ -1,2 +1,36 @@
 class Venue
+  attr_reader :name,
+              :capacity,
+              :patrons
+
+  def initialize(name, capacity)
+    @name     = name
+    @capacity = capacity
+    @patrons  = []
+  end
+
+  def add_patron(patron)
+    @patrons << patron
+  end
+
+  def yell_at_patrons
+    upcased_patrons = patrons.map do |patron|
+      patron.upcase
+    end
+    upcased_patrons
+  end
+
+  def over_capacity?
+    if @patrons.length <= @capacity
+      false
```

Comment:
Good use of `if/else` here. When would we be over_capacity? Could we just write that and since our method has a `?` we know that it will return a boolean. 


So it might look like:
```
@patrons.length > @capacity
```